static/app/components/events/eventReplay/replayClipPreview.tsx

```diff
@@ -107,6 +114,16 @@ function ReplayPreviewPlayer({
 
   return (
     <PlayerPanel>
+      {replayRecord && (
+        <ReplayCellNoPadding
+          key="session"
+          replay={replayRecord}
+          eventView={eventView}
+          organization={organization}
+          referrer="replay-header"
+          referrer_table="main"
```

Comment:
`main` here is specifically referring to our main replays list

Reply:
@michellewzhang what should the name be you think?

Reply:
maybe `referrer_table` --> `referrer_location`
and for this use case `referrer_location` = `replay_inline_clip_header` or something like that

also, i think for the `referrer` param replays does something like this:
`const referrer = getRouteStringFromRoutes(routes);`
so you could also reuse this pattern, but it's mostly for analytics i believe, so not a huge deal

Reply:
@michellewzhang If i rename the parameter it will mess up analytics, no? What if I just made the parameter optional? I honestly don't see why we need two different referrer values, seems odd.

Reply:
i'd be okay with making them optional.

i'm looking into it again and it looks like `referrer` is also used for the query: https://github.com/getsentry/sentry/blob/faed4110ea748b55120a1230a85a857b53607496/static/app/views/replays/replayTable/tableCell.tsx#L303

`referrer_table` is also used to determine which tab of the replay details we go to:
https://github.com/getsentry/sentry/blob/faed4110ea748b55120a1230a85a857b53607496/static/app/views/replays/replayTable/tableCell.tsx#L317

but they're both used for analytics here:
https://github.com/getsentry/sentry/blob/faed4110ea748b55120a1230a85a857b53607496/static/app/views/replays/replayTable/tableCell.tsx#L327